ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_sums_products_differences.py

```diff
@@ -329,3 +329,36 @@ def test_numpy_diff(
         x=x[0],
         axis=axis,
     )
+    
+    
+# ediff1d
+@handle_frontend_test(
+    fn_tree="numpy.ediff1d",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_num_dims=2,
+        max_num_dims=2,
+        min_dim_size=1,
+        max_dim_size=10,
+        valid_axis=True,
+        force_int_axis=True,
+    ),
+)
```

Comment:
You will need to generate `to_begin` and `to_end`. These can be either a single number or an array.